### PR TITLE
[v13] pull fixes from upstream

### PR DESCRIPTION
**Pull request by vishdha**
TASK ID: [ASANA](https://app.asana.com/0/1198992588740629/1198992588764632)

Changes:
[fix: make queries show searchfields](https://github.com/frappe/erpnext/pull/21685)

### CODE REVIEW

**Comment by hrwX** [comment]
@vishdha Not all the changes have been pulled in. If its an issue, just copy and paste it in your instance and raise the PR

File: erpnext/selling/doctype/customer/customer.py [hunk 1]
```diff
@@ -352,20 +352,20 @@ def get_customer_list(doctype, txt, searchfield, start, page_len, filters=None):
...
+			case when name like %s then 0 else 1 end,
+			case when customer_name like %s then 0 else 1 end,
+			name, customer_name limit %s, %s
 		""".format(match_conditions=match_conditions) % (", ".join(fields), searchfield, "%s", "%s", "%s", "%s", "%s", "%s"),
-			("%%%s%%" % txt, "%%%s%%" % txt, "%%%s%%" % txt, "%%%s%%" % txt, start, page_len))
+		("%%%s%%" % txt, "%%%s%%" % txt, "%%%s%%" % txt, "%%%s%%" % txt, start, page_len))
 
 
-def check_credit_limit(customer, company, reference_doctype, reference_document, ignore_outstanding_sales_order=False, extra_amount=0):
+def check_credit_limit(customer, company, ignore_outstanding_sales_order=False, extra_amount=0):
```
Comment (rvpasari) [inline erpnext/selling/doctype/customer/customer.py]: Please put in a detailed docstring explaining what this function does. Ensure you mention the arguments passed into it and what they represent (if any) and the values returned by it (if any).

Comment (vishdha) [reply erpnext/selling/doctype/customer/customer.py]: > @vishdha Not all the changes have been pulled in. If its an issue, just copy and paste it in your instance and raise the PR

there are 6 commit upstream. I have cherry-picked all of them. I dont think there is an extra commit.

**Comment by hrwX** [comment]
@vishdha Closing this, since there is only code formatting in the PR.

**Pull request closed without merging**